src/Serilog.Sinks.AzureDocumentDb/project.json

```diff
@@ -1,6 +1,6 @@
 {
     "title": "Serilog.Sinks.AzureDocumentDB",
-    "version": "3.5.17-*",
+    "version": "3.5.18-beta",
```

Comment:
This isn't necessary - on the `dev` branch, the version suffix will be added automatically in place of the `*`.